pkgs/tools/cd-dvd/cdimgtools/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, autoreconfHook, makeWrapper, libdvdcss, libdvdread, perl, perlPackages, asciidoc, xmlto, sourceHighlight, docbook_xsl, docbook_xml_dtd_45 }:
+stdenv.mkDerivation rec {
+  name = "cdimgtools-${version}";
+  version = "0.3";
+
+  src = fetchgit {
```

Comment:
I think you can use `fetchFromRepoOrCz`.

Reply:
done :-) thanks!